src/wallet/service.go

```diff
@@ -24,16 +24,20 @@ type BalanceGetter interface {
 // Service wallet service struct
 type Service struct {
 	sync.RWMutex
-	wallets        Wallets
-	firstAddrIDMap map[string]string // key: first address in wallet, value: wallet id
-
-	WalletDirectory string
+	wallets          Wallets
+	firstAddrIDMap   map[string]string // key: first address in wallet, value: wallet id
+	DisableWalletAPI bool
```

Comment:
If `DisableWalletAPI ` won't be used outside, we should make it private.

Reply:
Fixed.